libs/libs-pomfix/lib/fix.js

```diff
@@ -0,0 +1,196 @@
+var es = require('event-stream')
+var combine = require('stream-combiner')
+
+var notNull = function (o) { return o !== null }
+var notUndefined = function (o) { return o !== undefined }
+
+var processPom = function ($) {
+  var descriptor = $('plugin descriptor')
+    .map(function () {
+      var descriptor = $(this).text().trim()
+      var pattern = /mvn:(.*)\/(.*)\/(.*)\/xml\/features/
+      var match = descriptor.match(pattern)
+
+      if (match !== null) {
```

Comment:
I am looking at the rest of the code and I use the same pattern of `!== null` in if statements. The only reason I need a function, is for `Array.filter()`. So I actually think I want to leave it the same.